Фича "Лента событий"

goloviu opened the pull request:
feat: добавить ленту событий
test: добавить тесты для новых методов

anbananova committed:
feat: initial commit add-reviews feature

JuliaOlshevskaia committed:
feat: add recommendation films for user

JuliaOlshevskaia committed:
feat: delete '=='

JuliaOlshevskaia committed:
fix: delete 'drop table'

anbananova committed:
feat/refactor: added tests for ReviewService, ReviewDbStorage + rename checkValid

goloviu committed:
Фича "Лента событий"

feat: добавить ленту событий
test: добавить тесты для новых методов

goloviu committed:
Фича "Лента событий"

feat: добавить ленту событий
test: добавить тесты для новых методов

goloviu committed:
fix: удалить неиспользуемые импорты

goloviu committed:
fix: удалить неиспользуемые импорты

goloviu committed:
fix: удалить неиспользуемые импорты

goloviu committed:
fix: удалить неиспользуемые импорты

anbananova commented on src/main/java/ru/yandex/practicum/filmorate/storage/user/UserDbStorage.java:
```diff
@@ -95,6 +97,27 @@ public boolean isUserExist(Integer userId) {
+    }
+
+    @Override
+    public boolean saveUserFeed(Integer userId, Integer eventTypeId, Integer operationId, Integer entityId) {
```
не очень читаемы в коде айди operationId , entityId при вызове метода savedUserFeed. мне кажется, было бы более читаемо и нагляднее сделать для этих двух событий свои Enum классы перечислений. а внутри метода savedUserFeed использовать значение перечисление, вроде есть метод в них valueOf().

anbananova reviewed (changes requested):
все хорошо, можно зарефакторить использование событий ленты

JuliaOlshevskaia committed:
fix: rename userLikes, otherUserLikes

JuliaOlshevskaia committed:
fix: add checking exist user

JuliaOlshevskaia reviewed (approved):
Замечаний нет

goloviu committed:
Merge pull request #10 from goloviu/add-recommendations

feat: add recommendation films for user

goloviu committed:
fix: Заменить параметры метода saveUserFeed с получения eventId, operationId на enum
fix: Добавить плагин в мавен для декодинга в UTF-8 при maven test
fix: добавить CASCADE в sql скрипте

goloviu committed:
Фича "Лента событий"

fix: Заменить параметры метода saveUserFeed с получения eventId, operationId на enum
fix: Добавить плагин в мавен для декодинга в UTF-8 при maven test
fix: добавить CASCADE в sql скрипте

goloviu committed:
Фича "Лента событий"

fix: добавить пробел

goloviu committed:
Фича "Лента событий"

fix: убрать пробел

anbananova reviewed (approved):
все класс, спасибо
